packages/editor/src/editor-integration/create-basic-plugins.tsx

```diff
@@ -77,15 +83,27 @@ export function createBasicPlugins(props: Required<PluginsConfig>) {
       visibleInSuggestions: true,
       icon: <IconHighlight />,
     },
+    // Image plugin will have the `validate` prop by default, but if the user
+    // didn't provide the `upload` prop, Image plugin will not be created
+    ...(hasImagePluginConfigUploadPropertyDefined(config.image)
+      ? [
+          {
+            type: EditorPluginType.Image,
+            plugin: createImagePlugin(config.image),
+            visibleInSuggestions: true,
+            icon: <IconImage />,
+          },
+        ]
+      : []),
```

Comment:
We would need to add this to multimedia plugin (and soon image drag and drop) as well, right?

Reply:
Good point, I'll look into multimedia plugin for integrations.

I don't think that anything changes on the web/frontend side however, as the Image plugin there is initialized in the same way as it used to be.

Reply:
>Alright, the solution that comes to my mind is to remove the Image plugin from allowedPlugins in the Multimedia plugin config, if Image plugin upload config prop is not provided by the user of the Editor. What do you think about that?

Makes sense. but trying to integrate the editor without image support is also an edge case and I'd be also fine with an descriptive error message somewhere for example :)

Reply:
Added a condition for now, let's see how messy the edge cases get and consider error messages then. I wouldn't deal with error messages in this PR because, [here for example](https://github.com/serlo/frontend/blob/staging/packages/editor/src/plugins/image/index.ts#L96C50-L96C65), we still use the web/frontend code in the editor. That should be cleaned first, before any more error messages, I think.

Thank you for noticing the Multimedia edge case! For the Dropzone plugin (I prefer to use a one-work name until we get a real name from @GregorZupan), it's easier, we just disable it completely if Image upload isn't provided.